<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageLines</class>
 <widget class="QDialog" name="ManageLines">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>添加</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/icon/add.png</normaloff>:/icon/icon/add.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tabAddLine">
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <attribute name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icon/icon/subway.png</normaloff>:/icon/icon/subway.png</iconset>
    </attribute>
    <attribute name="title">
     <string>线路(&amp;L)</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>560</width>
       <height>160</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;提示：&lt;/span&gt;新增线路方法&lt;/p&gt;&lt;p&gt;1.先添加新线路信息；&lt;/p&gt;&lt;p&gt;2.然后在工具【站点】中添加线路站点；&lt;/p&gt;&lt;p&gt;3.最后在工具【连接】中确定站点连接关系；&lt;/p&gt;&lt;p&gt;4.也可以用文本方式简单快捷添加，格式需规范。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>线路名称(&amp;N)：</string>
     </property>
     <property name="buddy">
      <cstring>lineEditLineName</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLineName">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入线路名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddLine">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>320</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>添加线路</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>线路颜色(&amp;C)：</string>
     </property>
     <property name="buddy">
      <cstring>pushButtonChooseColor</cstring>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonChooseColor">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>拾取颜色</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserAddLine">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>200</y>
       <width>140</width>
       <height>60</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>180</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>添加信息：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAddStation">
    <attribute name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icon/icon/station.png</normaloff>:/icon/icon/station.png</iconset>
    </attribute>
    <attribute name="title">
     <string>站点(&amp;S)</string>
    </attribute>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxLatitude">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>88</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>85</width>
       <height>20</height>
      </size>
     </property>
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>30.899999999999999</double>
     </property>
     <property name="maximum">
      <double>31.449999999999999</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>30.899999999999999</double>
     </property>
    </widget>
    <widget class="QLabel" name="labelLatitude">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>北纬&amp;N</string>
     </property>
     <property name="buddy">
      <cstring>doubleSpinBoxLatitude</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="labelLongitude">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>东经&amp;E</string>
     </property>
     <property name="buddy">
      <cstring>doubleSpinBoxLongitude</cstring>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxLongitude">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>85</width>
       <height>20</height>
      </size>
     </property>
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>121.000000000000000</double>
     </property>
     <property name="maximum">
      <double>122.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>121.000000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditStationName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>添加站点名</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserLinesSelected">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>站点名称(&amp;N)：</string>
     </property>
     <property name="buddy">
      <cstring>lineEditStationName</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>地理坐标(&amp;P)：</string>
     </property>
     <property name="buddy">
      <cstring>doubleSpinBoxLongitude</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>已选择：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddStation">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>330</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>添加站点</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>选择 添加到线路：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>200</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>添加信息：</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserAddStaiton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>230</y>
       <width>140</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;有效范围:&lt;/p&gt;&lt;p&gt;min [ 121.0 , 30.9 ]&lt;/p&gt;&lt;p&gt;max [ 122.0 , 31.5 ]&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>200</width>
       <height>270</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAddConnection">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icon/icon/connect.png</normaloff>:/icon/icon/connect.png</iconset>
    </attribute>
    <attribute name="title">
     <string>连接(&amp;C)</string>
    </attribute>
    <widget class="QComboBox" name="comboBoxConnectStation1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>站点1(&amp;O)：</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxConnectStation1</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>站点2(&amp;T)：</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxConnectStation2</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConnectStation2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>所属线路(&amp;L)：</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxConnectLine</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConnectLine">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>560</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;提示：&lt;/span&gt;连接站点&lt;/p&gt;&lt;p&gt;1.连接“站点1”和“站点2”为一条边；&lt;/p&gt;&lt;p&gt;2.“站点1”和“站点2”属于同一条线路中的不同站点；&lt;/p&gt;&lt;p&gt;3.站点及线路选择可以选择也可直接输入。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAddByText">
    <attribute name="title">
     <string>文本方式</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>560</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddByText">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>330</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
